backstory/src/main/java/com/google/sps/story/data/StoryEndingTools.java

```diff
@@ -0,0 +1,106 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.story.data;
+
+/**
+ * Static methods to help end a story naturally
+ */
+public class StoryEndingTools {
+  /** the endings to add to the story (should be one sentence) */
+  private static String[] ENDINGS = {
+      "The End.",
+      "They lived happily ever after.",
+      "Then, everything went horribly wrong.",
+      "And--- that's a wrap.",
+      "Goodbye!",
+      "Then the director screamed \"CUT!\".",
+      "With that, our story draws to a close.",
+      "We'll never know what happened next.",
+  };
+
+  /**
+   * Overrides default constructor to ensure class can't be instantiated.
+   */
+  private StoryEndingTools() {
+    throw new AssertionError();
+  }
+
+  /**
+   * Takes a story from GPT-2 and returns onee with a more natural ending.
+   *
+   * @param story the story to return with a natural ending
+   * @return story with ending
+   * @throws IllegalArgumentException if story is null
+   */
+  public static String endStory(String story) {
+    if (story == null) {
+      throw new IllegalArgumentException("Story should not be null");
+    }
```

Comment:
I'm happy to make a change, but a bit confused. I don't believe I trim whitespace in this file. What is "trimming whitespace"  referring to in this comment? @ykjchen

Reply:
Like this:
```
if (story == null || story.trim().length() == 0) { ...
```

This would ensure that this is not just an empty string.

https://www.geeksforgeeks.org/java-string-trim-method-example/#:~:text=trim()is%20a%20built,and%20returns%20the%20omitted%20string.

Reply:
That makes sense. Will do.

Reply:
Done